keps/sig-node/4800-cpumanager-split-l3cache/README.md

```diff
@@ -0,0 +1,680 @@
+
+- [x] Feature gate (also fill in values in `kep.yaml`)
+  - Feature gate name: `CPUManagerAlphaPolicyOptions`
+  - Components depending on the feature gate: `kubelet`
+- [x] Other
+  - Describe the mechanism: Change the `kubelet` configuration to set a `CPUManager` policy of static then setting the policy option of `align-cpus-by-uncorecache`
+  - Will enabling / disabling the feature require downtime of the control
+    plane? No.
+  - Will enabling / disabling the feature require downtime or reprovisioning
+    of a node? Yes, a `kubelet` restart is required for changes to take place.
+
+###### Does enabling the feature change any default behavior?
+
+No, to enable this feature, it must be explicitly set in the `CPUManager` static policy and the policy option `align-cpus-by-uncorecache` must be set.
+
+###### Can the feature be disabled once it has been enabled (i.e. can we roll back the enablement)?
+
+Yes this feature can be disabled it will just require a restart of `kubelet`. The Kubelet configuration will need to be set with the static policy option and align-cpus-by-uncorecache flag removed.
+
+
+###### What happens if we reenable the feature if it was previously rolled back?
+
+Feature will be enabled. Proper drain of node and restart of kubelet required. Feature is not intended to be enabled/disabled dynamically, similar to static policy.
+
+###### Are there any tests for feature enablement/disablement?
+
+Option is not enabled dynamically. To enable/disable option, cpu_manager_state must be removed and kubelet must be restarted.
+Unit tests will be implemented to test if the feature is enabled/disabled.
+E2e node serial suite can be use to test the enablement/disablement of the feature since it allows the kubelet to be restarted.
+
+
+### Rollout, Upgrade and Rollback Planning
+
+<!--
+This section must be completed when targeting beta to a release.
+-->
+
+###### How can a rollout or rollback fail? Can it impact already running workloads?
+
+Kubelet restarts are not expected to impact existing CPU assignments to already running workloads
+
+
+###### What specific metrics should inform a rollback?
+
+N/A
+
+###### Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?
+
+N/A
+
+
+###### Is the rollout accompanied by any deprecations and/or removals of features, APIs, fields of API types, flags, etc.?
+
+N/A
+
+
+### Monitoring Requirements
+
+Include CacheID info in podresources API to be able to verify assignment.
+
+###### How can an operator determine if the feature is in use by workloads?
+
+Reference podresources API to determine CPU assignment and CacheID assignment.
+
+###### How can someone using this feature know that it is working for their instance?
+
+Reference podresources API to determine CPU assignment and CacheID assignment.
```

Comment:
it is probably more broader question about other static policy options like `full-pcpu` alignment, to see if criteria really met?